solr/solrj/src/java/org/apache/solr/client/solrj/request/DirectXmlRequest.java

```diff
@@ -55,8 +55,13 @@ public SolrParams getParams() {
   }
 
   @Override
-  public String getRequestType() {
-    return SolrRequestType.UPDATE.toString();
+  protected SolrRequestType getBaseRequestType() {
```

Comment:
[0] Unrelated to this PR, but it's a shame that DirectXmlRequest isn't a child of either `AbstractUpdateRequest` or `UpdateRequest`.  Anyone remember why this is by chance?

Reply:
IMO we should deprecate `DirectXmlRequest`.  It's unused, untested, and not needed.  I'll do so in my deprecation branch/PR

Reply:
I already did that: https://github.com/apache/solr/blob/cbc2321598f30066dab4ba396e6c593577f9d8f4/solr/solrj/src/java/org/apache/solr/client/solrj/request/DirectXmlRequest.java#L33
I should merge that PR so we can more easily communicate to even each other what things are going away so we shouldn't spend attention on